plugin/definitive.vim

```diff
@@ -52,7 +52,9 @@ function! definitive#FindDefinition(...)
   let l:match_in_current_file = search(l:search_text, 'wcbs')
 
   if l:match_in_current_file
-    exec l:match_in_current_file
+    call cursor(l:match_in_current_file, 0)
+    let [_, col] = searchpos(l:wanted_definition, 'c', line('.'))
+    call cursor(0, col)
```

Comment:
I'm fine with doing this but I think it could be done with one call like
`call searchpos(l:wanted_definition, 'c', l:match_in_current_file)`

Reply:
Also would be nice if you could have it do this when the match is not in the current file but I won't require it for this PR

Reply:
@misterbuckley sorry for the late response, and thanks for merging! :raised_hands: 

I didn't realize the search functions already move the cursor, thanks for simplifying that.

> Also would be nice if you could have it do this when the match is not in the current file but I won't require it for this PR

Ah yes I wanted to add this but forgot about it, submitted https://github.com/misterbuckley/vim-definitive/pull/18 for this now.